.github/workflows/codecoverage.yml

```diff
@@ -0,0 +1,58 @@
+name: Code Coverage (Apollo tests)
+on:
+  schedule:
+    - cron: "0 0 * * 6"
```

Comment:
Can we configure it to send an e-mail to blockchain-concord-team@vmware.com every time it runs?

Reply:
@teoparvanov , great idea! However, I suggest accessing the email address via GITHUB secrets. We do not want to expose the email address to the world.

Reply:
I have added a action job which sends mail to every user. 
Only thing is left to configure github secret keys in VMware Concord-bft official account.
I will talk to Robert regarding same.